frontend/src/metabase/databases/utils/settings.ts

```diff
@@ -0,0 +1,8 @@
+import type { Database } from "metabase-types/api";
+
+export function hasDatabaseTableEditingEnabled(_database: Database) {
```

Comment:
Should this be on `PLUGIN_DATA_EDITING`? That way if they don't have the feature enabled we can default the implementation to false?

I'm wondering about the case where a user has configured a database while paying for the enterprise edition but is later downgraded to OSS. Maybe the BE already takes care of this, but we'll want to make sure that the `true` isn't respected. (Also +1 to Ryan's point in `frontend/src/metabase-lib/v1/metadata/Database.ts`, would be nice to collocate the logic).